awxkit/awxkit/cli/custom.py

```diff
@@ -41,13 +41,13 @@ def resource(self):
     def perform(self):
         raise NotImplementedError()
 
-    def add_arguments(self, parser):
+    def add_arguments(self, parser, resource_options_parser):
```

Comment:
what's the difference between the `parser` and `resource_options_parser`? Does this only apply to the custom actions?

Reply:
`parser` is a Python `argparse.ArgumentParser` object
`resource_options_parser` is `awxkit.cli.options.ResourceOptionsParser`, the class we use to parse HTTP Options.